### PR TITLE
Add quick start documentation to google cloud storage.

### PR DESCRIPTION
**Pull request by quoctruong**
This is from issue https://github.com/GoogleCloudPlatform/google-cloud-powershell/issues/312

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/380)

### CODE REVIEW

File: website/js/app/controllers/content-product.ng [hunk 1]
```diff
@@ -115,6 +115,26 @@ Write-GcsObject -Bucket $bucket -File "test-file.png" -ObjectName "test/test.png
 
 # Download a GCS object to disk.
 Read-GcsObject $bucket "object-name" -OutFile "output-file.png"</pre>
+    <h2>Cloud Storage PowerShell Provider</h2>
+    <p>
+      Cloud Tools for PowerShell includes a PowerShell provider for Google Cloud Storage.
+      This provider allows you to use commands like cd, dir, copy and del to navigate
```
Comment (chrsmith) [inline website/js/app/controllers/content-product.ng]: Let's used a fixed-width font for terminal commands. (That is surround them with "\<code\>" tags.)

like \<code\>cd\</code\>, \<code\>dir\</code\>, ...

File: website/js/app/controllers/content-product.ng [hunk 2]
```diff
@@ -115,6 +115,26 @@ Write-GcsObject -Bucket $bucket -File "test-file.png" -ObjectName "test/test.png
...
+      and manipulate your data in Cloud Storage as if the data were on a local file system.
+
+      To directly use the provider, you can start Cloud Tools for PowerShell using the shortcut
+      from the start menu. This will launch a PowerShell console with the provider loaded:
+    </p>
+    <pre>
+# Navigate to Google Cloud Storage
+cd gs:\
+# Show the available buckets
+ls
```
Comment (chrsmith) [inline website/js/app/controllers/content-product.ng]: "dir" is more Windows-idiomatic.

File: website/js/app/controllers/content-product.ng [hunk 3]
```diff
@@ -115,6 +115,26 @@ Write-GcsObject -Bucket $bucket -File "test-file.png" -ObjectName "test/test.png
...
+      This provider allows you to use commands like cd, dir, copy and del to navigate
+      and manipulate your data in Cloud Storage as if the data were on a local file system.
+
+      To directly use the provider, you can start Cloud Tools for PowerShell using the shortcut
+      from the start menu. This will launch a PowerShell console with the provider loaded:
+    </p>
+    <pre>
+# Navigate to Google Cloud Storage
+cd gs:\
+# Show the available buckets
```
Comment (chrsmith) [inline website/js/app/controllers/content-product.ng]: Blank vertical line between sections for readability?

File: website/js/app/controllers/content-product.ng [hunk 4]
```diff
@@ -115,6 +115,26 @@ Write-GcsObject -Bucket $bucket -File "test-file.png" -ObjectName "test/test.png
...
+ls
+# Create a new bucket
+mkdir my-new-bucket</pre>
+    <p>
+      You can also make the provider available in any PowerShell session by importing
+      the Cloud Tools for PowerShell module via "Import-Module GoogleCloud".
```
Comment (chrsmith) [inline website/js/app/controllers/content-product.ng]: I'd use the \<code\> tag for this too, instead of quotes.

**Comment by quoctruong** [comment]
Added the <code> tags. Thanks!

**Pull request merged**